libs/images/sonar-scanner/README.md

```diff
@@ -21,8 +28,31 @@ export SONAR_PROJECT_KEY=my-app \
 export SONAR_ORGANIZATION=my-org
 ```
 
+Run with sonar scanner for python nodejs ruby projects
+```
+docker run -e SONAR_HOST_URL=https://sonarcloud.io -e SONAR_TOKEN=$SONAR_TOKEN -e SONAR_PROJECT_KEY=$SONAR_PROJECT_KEY -e \
+SONAR_ORGANIZATION=$SONAR_ORGANIZATION -e BUILD_NUMBER=1.2.3 --rm -it -v $(pwd):/usr/src amidostacks/ci-sonarscanner:latest
+```
+
+Run with dotnet projects. example at a solution level
+```bash 
+cd $GITHUB_AMIDO_STACKS_DOTNET/src/api
+docker run -v $(pwd):/usr/api --rm -it amidostacks/ci-sonarscanner:latest /bin/bash
+dotnet sonarscanner begin /k:"dotnet-api" /d:sonar.host.url=https://sonarcloud.io /d:sonar.login=my_token /d:sonar.cs.opencover.reportsPaths="**/coverage.opencover.xml" /o:"my_org"
+dotnet clean
+dotnet restore
```

Comment:
Remove, build does restore automagically.

Reply:
it's just a readme but ok - I had some trouble doing just build inside a container - especially if you mount local dir to a container.
the pipeline task won't need it yes you are right though